server/protocols/jmap-rfc-8621/src/main/scala/org/apache/james/jmap/mail/MailboxGet.scala

```diff
@@ -32,7 +32,9 @@ case class MailboxGetRequest(accountId: AccountId,
                              ids: Option[Ids],
                              properties: Option[Properties])
 
-case class NotFound(value: List[MailboxId])
+case class NotFound(value: List[MailboxId]) {
+  def merge(other: NotFound): NotFound = NotFound(this.value ++ other.value)
```

Comment:
Should it be a set ? do we mind for duplicates ?

Reply:
Is a duplicate even possible? But if there is even a tiny possibility, we should enforce it then yes. Good remark !